frontend/src/components/Board/Readme.js

```diff
@@ -5,10 +5,7 @@ import _JSXStyle from 'styled-jsx/style'
 
 const Readme = ({ renderedReadme }) => (
   <div id="readme">
-    <div
-      // eslint-disable-next-line react/no-danger
-      dangerouslySetInnerHTML={{ __html: renderedReadme }}
-    />
+    <div dangerouslySetInnerHTML={{ __html: renderedReadme }} />
```

Comment:
I think this one is the one exception where I would suggest keeping the rule and using the `eslint-disable`, even though we use it in three places. We really need to know what we are doing with `dangerouslySetInnerHTML` and should avoid it if possible. We should have that extra check from eslint if someone tries to use it somewhere else.

Reply:
Resolved via 9b06911.